src/dropbox-base.js

```diff
@@ -205,6 +250,9 @@ export class DropboxBase {
     if (!['code', 'token'].includes(authType)) {
       throw new Error('Authorization type must be code or token');
     }
+    if (!['legacy', 'offline', 'online'].includes(tokenAccessType)) {
```

Comment:
Can we make this array a constant? The same for auth type (even though that's existing code it would be nice).

Reply:
Added in next revision